activerecord/lib/active_record/fixture_set/table_row.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  class FixtureSet
+    class TableRow # :nodoc:
+      def initialize(fixture, table_rows:, label:, now:)
+        @table_rows = table_rows
+        @label = label
+        @now = now
+        @row = fixture.to_hash
+        fill_row_model_attributes
+      end
+
+      def to_hash
+        @row
+      end
+
+      private
+
+        def model_metadata
+          @table_rows.model_metadata
+        end
+
+        def model_class
+          @table_rows.model_class
+        end
+
+        def fill_row_model_attributes
+          return unless model_class
+          fill_timestamps
+          interpolate_label
+          generate_primary_key
+          resolve_enums
+          @table_rows.resolve_sti_reflections(@row)
+        end
+
+        def reflection_class
+          @refelction_class ||= if @row.include?(model_metadata.inheritance_column_name)
```

Comment:
Little typo here `@refelction_class` -> `@reflection_class`

Reply:
🤦‍♂️ Thanks, I'll fix that!